sdk/init.go

```diff
@@ -265,7 +265,7 @@ func Init(
 	dialAddr string,
 	token string,
 ) error {
-	client, err := ethclient.Dial(dialAddr)
+	client, err := connectEthClient(dialAddr, token)
```

Comment:
@jimpick for consistency i'd just use the `connectEthClient` helper method in `sdk/extern.go` so that we share instantiation code for the eth client in case we observe some funky behavior

Reply:
I don't understand your comment. Isn't that exactly what the code is doing there?